templates/terraform/env_var_context.go.erb

```diff
@@ -17,5 +17,7 @@
 		"<%= var_name -%>": getTestProjectFromEnv(),
 	<% elsif var_type == :FIRESTORE_PROJECT_NAME -%>
 		"<%= var_name -%>": getTestFirestoreProjectFromEnv(t),
+	<% elsif var_type == :CUST_ID  -%>
```

Comment:
I think you also need to add something to provider/terraform/examples.rb to get the cust_id to appear in the docs example